### PR TITLE
Update files 

### PR DESCRIPTION
**Pull request by touredri**
- [x] index files change
- [ ] css files change
- [ ] add .gitigore file
- [ ] add workflow
- [ ] add linters

### CODE REVIEW

File: index.htlm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 <body>
   <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
-  <nav id="navbar">
+  <nav id="navbar" class="navbar">
```
Comment (touredri) [inline index.htlm:14]: Please put the nav inside the header and by doing this you can manage correctly the nav with other elements inside using CSS flexbox

File: index.htlm [hunk 2]
```diff
@@ -37,7 +38,7 @@
       <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
       <code>It is a long established fact that a reader will be distracted by the</code>
       <ul>
-        <li>using Lorem Ipsum is that it has a more-or-less </li>
+        <li>using Lorem Ipsum is that it has a more-or-less lrud sur uertps </li>
```
Comment (touredri) [inline index.htlm:41]: I think it would be better to not use dummy text and provide meaningful content. This could improve the code because it allows an understanding of what to put thereafter.

File: style.css [hunk 3]
```diff
@@ -20,7 +20,7 @@
 #navbar > header > a {
```
Comment (touredri) [inline style.css:20]: Here you can take #navbar > a; because the navigator will be quick to find the a tag inside and apply properties

**Review by touredri — COMMENTED** [review]
Hi @touredri 
### ♻️ Required Changes ♻️



Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on.

### Highlights
- [ ] Correct Github flow ✔️
- [ ] Descriptive PR ✔️

### Required Changes ♻️
- [ ]  Try to correct all linter problems 
- [ ] Use Microverse's readme template to make the best description for your project
- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

File: index.htlm (hunk 1 above)
Comment (anagudelogu) [reply index.htlm:14]: - [ ] Great job asking politely for the change! 💯 However, a good review should also contain an explanation of "why" you think is a good idea to implement this change. Please consider editing this comment to include this explanation! 🤓 🔧 

>This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
>
>This request lacks a "why": Why don't you use an article tag here instead of a div?

File: index.htlm (hunk 2 above)
Comment (anagudelogu) [reply index.htlm:41]: - [ ] I think that you could rephrase this to be more clear and add an explanation as well so people reading your suggestion can easily understand how the code would improve. For example, it could be rephrased like this: `I think it would be better to not use dummy text and provide meaningful content. This could improve the code because ... [Add the explanation here]`. Please edit this comment in order to be clear and include the explanation! 🤓 🔧 

>This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
>
>This is a vague comment: *The README is not descriptive, why don't you make it better?*

File: style.css (hunk 3 above)
Comment (anagudelogu) [reply style.css:20]: - [ ] Once again, please consider providing an explanation! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @touredri, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub's review features! 💯 
✅ Great review summary. 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ ] **[OPTIONAL]:** I think it would be great if you use emojis in your comments. They make the code review more human-friendly 👽🤖.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @touredri , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- Good use of the inline comment.✔
- Polite commentrs. ✔
- GitHub flow is used correctly. ✔


## Optional suggestions
- [ OPTIONAL ] You can also share sources to help the student understand the topic better. 📗

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ABDULALI3468 — APPROVED** [review]
Hi @touredri , 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
